README.md

```diff
@@ -62,7 +62,7 @@ rustup run nightly cargo bench
 Decoding is aided by some pre-calculated tables, which are generated by:
 
 ```
-cargo run --bin make_decode_table > src/decode_tables.rs.tmp && mv src/decode_tables.rs.tmp src/decode_tables.rs
```

Comment:
Since the table tool isn't something that should be installed if you `cargo install base64` I moved it to examples where it won't affect lib consumers (or installers) at all.